napari/viewer.py

```diff
@@ -18,3 +18,10 @@ def __init__(self, title='napari'):
         self.window = Window(qt_viewer)
         self.screenshot = self.window.qt_viewer.screenshot
         self.camera = self.window.qt_viewer.view.camera
+
+
+@Viewer.bind_key('v')
```

Comment:
i feel like this should go in the `qt_viewer.py` file, not here - right @kne42?

Reply:
The layer shortcuts are all on the models... But you might be right!